packages/@okta/vuepress-theme-prose/const/home.const.js

```diff
@@ -1,109 +1,110 @@
-export const tileData = [
-  {
-    name: "Web App",
-    categoryLink: "/docs/guides/sign-into-web-app/aspnet/before-you-begin/",
-    items: [
```

Comment:
These were moved to the "home" `.md` file and this file was removed. You will have to re-adjust over there.

Reply:
Thanks for pointing out...

Reply:
try pulling again -- you should see https://github.com/okta/okta-developer-docs/commit/7b1fcdcf75c511ce67389d4c0cf9ab0e89e9e6ca as the tip of your `master` branch